GPU/D3D11/FramebufferManagerD3D11.cpp

```diff
@@ -105,6 +105,18 @@ FramebufferManagerD3D11::FramebufferManagerD3D11(Draw::DrawContext *draw)
 	vb.Usage = D3D11_USAGE_DYNAMIC;
 	vb.CPUAccessFlags = D3D11_CPU_ACCESS_WRITE;
 	device_->CreateBuffer(&vb, nullptr, &quadBuffer_);
+
+	D3D11_TEXTURE2D_DESC packDesc{};
+	packDesc.CPUAccessFlags = D3D11_CPU_ACCESS_READ;
+	packDesc.BindFlags = 0;
+	packDesc.Width = 512;  // 512x512 is the maximum size of a framebuffer on the PSP.
```

Comment:
I don't think this is true.  This is the maximum size that can be textured from, but I think drawn can be wider.  See #4739.  Also some games (I believe Silent Hill?) draw to one big 1024 wide surface, but draw on the left and right sides - see curRTOffsetX.

I do think 512 may be the max height.  I've never seen wider than 1024, so that may be the max width.

Also, I don't know about HD remasters.  Presumably they draw double the width and height or something?

-[Unknown]